library/src/main/java/com/novoda/downloadmanager/lib/DownloadStatus.java

```diff
@@ -13,6 +13,10 @@
 final class DownloadStatus {
 
     /**
+     * This download has been marked for deletion and it will be deleted in the future
+     */
+    public static final int DELETING = 188;
```

Comment:
sleepy Alex, this is a status code like `public static final int SUBMITTED = 189;` i assume